packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -86,14 +86,14 @@ function NetworkListbox({
     const { isArbitrum, isArbitrumNova } = isNetwork(value.chainID)
 
     if (!isArbitrum) {
-      return 'bg-[rgba(118,121,145,0.8)]'
```

Comment:
this is a different colour from Figma

it should be this var

https://github.com/OffchainLabs/arbitrum-token-bridge/blob/75f485412716be388b779a2f32c611db9bc3e912/packages/arb-token-bridge-ui/tailwind.config.js#L16